components/ReadEnv.coffee

```diff
@@ -17,15 +17,14 @@ exports.getComponent = ->
     datatype: 'object'
     required: false
 
-  noflo.helpers.WirePattern c,
-    in: 'key'
-    out: 'out'
-    forwardGroups: true
-    async: true
-  , (data, groups, out, callback) ->
-    value = process.env[data]
-    return callback new Error "No environment variable #{data} set" if value is undefined
-    out.send process.env[data]
-    do callback
+  c.forwardBrackets =
+    key: ['out', 'error']
 
-  c
+  c.process (input, output) ->
+    data = input.get 'key'
+    value = process.env[data]
```

Comment:
Fixed in https://github.com/noflo/noflo-core/pull/78/commits/1bf14669f396a6437c98931d9bd04b47e9c8e55b

Was a bad test.